handler/src/main/java/io/netty/handler/timeout/IdleStateEvent.java

```diff
@@ -21,19 +21,14 @@
  * A user event triggered by {@link IdleStateHandler} when a {@link Channel} is idle.
  */
 public final class IdleStateEvent {
-    public static final IdleStateEvent FIRST_READER_IDLE_STATE_EVENT = new IdleStateEvent(IdleState.READER_IDLE, true);
```

Comment:
@trustin - The alternative approach you suggest in https://github.com/netty/netty/pull/5164#issuecomment-213276989 lgtm!